docs/src/docs/next-steps.md

```diff
@@ -0,0 +1,232 @@
+# Next Steps with Kit
+
+In this guide you'll learn how to:
+* Make your own Kitfile
+* The power of `unpack`
+* Read the Kitfile or manifest from a ModelKit
+* Tag ModelKits and keep your registry tidy
+
+## Making your own Kitfile
+
+A Kitfile is the configuration document for your ModelKit. It's written in YAML so it's easy to read. There are four main parts to a Kitfile:
+1. ModelKit metadata in the `package` section, including the author, description, and license
+1. Path to the Jupyter notebook folder in the `code` section
+1. Path to the datasets in the `datasets` section (you can have multiple datasets in the same page)
+1. Path to the serialized model in the `model` section
+
+A Kitfile only needs the `package` section, plus one of the other sections.
+
+The `datasets` and `code` sections can include multiple assets, but the `model` section can only contain a single model (you can chain models by using multiple ModelKits).
+
+Here's an example Kitfile:
+
+```yaml
+manifestVersion: v1.0.0
+
+package:
+  authors:
+  - Jozu
+  description: Small language model based on Mistral-7B fine tuned for answering film photography questions.
+  license: Apache-2.0
+  name: FilmSLM
+
+model:
+  description: Film photography Q&A model using Mistral-7B
+  framework: Mistral-7B
+  license: Apache-2.0
+  name: FilmSLM
+  path: ./models/film_slm:champion
+  version: 1.2.6
+
+datasets:
+- description: Forum postings from sites like rangefinderforum, DPreview, PhotographyTalk, and r/AnalogCommunity
+  name: training data
+  path: ./data/forum-to-2023-train.csv
+- description: validation data
+  name: validation data
+  path: ./data/test.csv
+
+code:
+- description: Jupyter notebook with model training code in Python
+  path: ./notebooks
+```
+
+A minimal ModelKit for distributing a pair of datasets might look like this:
+```yaml
+manifestVersion: v1.0.0
+
+datasets:
+- name: training data
+  path: ./data/train.csv
+- description: validation data
+  name: validation data
+  path: ./data/validate.csv
+```
+
+Right now you can only build ModelKits from files on your local system...but don't worry we're already working towards allowing you to [reference remote files](https://github.com/jozu-ai/kitops/issues/85). For example, building a ModelKit from a local notebook and model, but a dataset hosted on DvC, S3, or anywhere else.
+
+Once you've authored a Kitfile for your AI/ML project you can pack it up and store it in your local or remote repository.
+
+```sh
+kit pack . -t film-slm:champion
+kit push docker.io/jozubrad/film-slm:champion
```

Comment:
Needs to be tagged with the registry reference before push
`kit pack . -t docker.io/jozubrad/film-slm:champion`